tiny_tokenizer/word_tokenizers/kytea_tokenizer.py

```diff
@@ -22,12 +22,21 @@ def tokenize(self, text: str):
 
         if self.with_postag:
             response = self.kytea.getTagsToString(text)
-            response = response.replace("  ", " <SPACE>")  # FIXME
+            response = response.replace("\\ ", "<SPACE>")  # FIXME "私 は猫"
+            response = response.replace("  ", " <SPACE>")
```

Comment:
[Nits] response = response.replace(...).replace(...).replace(...) is simple I think.

Reply:
I update my code to use a method chaining.